docs/docsite/rst/guide_migration.rst

```diff
@@ -28,4 +31,331 @@ Module ``win_iis_website``
 
 Migrated to :ref:`microsoft.iis.website <ansible_collections.microsoft.iis.website_module>`.
 
-To complete.
+community.windows.win_iis_website has been split into two modules:
+  * ansible.microsoft.iis.website.
+  * ansible.microsoft.iis.website_info.
+
+Custom site Parameters from string ($parameters) removed from the module
+win_iis_webbinding was consolidated into microsoft.iis.website
+Parameters bind_ip, bind_port, bind_hostname moved into $bindings ip, port, hostname
+$bindings now include add/set/remove functionality
+Updated from ``Ansible.ModuleUtils.Legacy`` to ``-CSharpUtil Ansible.Basic``
+
+.. code-block:: yaml
+
+  - name: Create a default website in 'Started' state
+    microsoft.iis.website:
+      name: WebSite
+      physical_path: C:\inetpub\wwwroot
+      state: started
+      bindings:
+        set:
+          - ip: 127.0.0.1
+            port: 80
+            hostname: my-website.com
+
+  - name: Create a website with two bindings, set all values and start it
+    microsoft.iis.website:
+      name: WebSite
+      site_id: 2
+      state: started
+      physical_path: C:\wwwroot\websites\my-website
+      application_pool: DefaultAppPool
+      bindings:
+        set:
+          - ip: 127.0.0.1
+            port: 8081
+            hostname: my-website.com
+          - ip: 127.0.0.2
+            port: 8082
+            hostname: my-website.net
+            protocol: https
+            use_sni: true
+            use_ccs: false
+            certificate_hash: 5409124040FA1C8FA74939BDA2EF8FD5975BD25B
+            certificate_store_name: MY
+
+  - name: Remove all the bindings from an existing website
+    microsoft.iis.website:
+      name: WebSite
+      bindings:
+        set: []
+
+  - name: Remove a binding from an existing website
+    microsoft.iis.website:
+      name: WebSite
+      bindings:
+        remove:
+          - ip: 127.0.0.1
+            port: 8081
+            hostname: my-website.com
+
+  - name: Add bindings to an existing website
+    microsoft.iis.website:
+      name: WebSite
+      bindings:
+        add:
+          - ip: 127.0.0.3
+            port: 8083
+            hostname: new-website.com
+          - ip: 127.0.0.4
+            port: 8084
+            hostname: new-website.net
+
+  - name: Stop a website
+    microsoft.iis.website:
+      name: WebSite
+      state: stopped
+
+  - name: Restart a website (non-idempotent)
+    microsoft.iis.website:
+      name: WebSite
+      state: restarted
+
+  - name: Change a website application pool
+    microsoft.iis.website:
+      name: WebSite
+      application_pool: NewAppPool
+
+  - name: Change a website physical path
+    microsoft.iis.website:
+      name: WebSite
+      physical_path: C:\wwwroot\websites\my-website
+
+Module ``website_info``
```

Comment:
No need for a separate section for `website_info`, it should be put together with the migration from `community.windows.win_iis_website`.